uspec/config_module_spec.rb

```diff
@@ -60,24 +62,24 @@ module FalseNil
 
 module MultipleExample
   extend ConfigModule
-  config_file './config/example.yml'
+  config_file "./config/example.yml"
   namespace :production, :dictionary
 end
 
-spec 'multiple namespaces can be set' do
+spec "multiple namespaces can be set" do
   MultipleExample.configuration == ExampleConfig.dictionary.configuration
 end
 
 module InvalidNamespaceExample
   extend ConfigModule
-  config_file './config/example.yml'
+  config_file "./config/example.yml"
   namespace :jimmy, :pop, :ali
 end
 
-spec 'incorrect namespaces raise informative errors' do
+spec "incorrect namespaces raise informative errors" do
   begin
     InvalidNamespaceExample.whatever
-  rescue => error
+  rescue StandardError => error
```

Comment:
This should have identical functionality?

Also I probably should have just done this:
```suggestion
  rescue ConfigModule::InvalidNamespaceError
    true
```

Reply:
Rubocop was complaining about the missing class. But yes, that looks better.